src/components/job-common.js

```diff
@@ -2,76 +2,120 @@ import { Fragment } from 'react'
+  return label + (count === undefined ? '' : `: ${count}`)
+}
+
+export const statusType = {
+  succeeded: {
+    id: 'succeeded',
+    label: _ => 'Succeeded',
+    icon: style => icon('check', { size: iconSize, style: { color: colors.success(), ...style } })
+  },
+  failed: {
+    id: 'failed',
+    label: _ => 'Failed',
+    icon: style => icon('warning-standard', { size: iconSize, style: { color: colors.danger(), ...style } })
+  },
+  running: {
+    id: 'running',
+    label: _ => 'Running',
+    icon: style => icon('sync', { size: iconSize, style: { color: colors.dark(), ...style } })
+  },
+  submitted: {
+    id: 'submitted',
+    label: _ => 'Submitted',
+    icon: style => icon('clock', { size: iconSize, style: { color: colors.dark(), ...style } })
+  },
+  waitingForCloudQuota: {
+    id: 'waitingForQuota',
+    label: _ => 'Submitted, Awaiting Cloud Quota',
+    icon: style => icon('error-standard', { size: iconSize, style: { color: colors.warning(), ...style } }),
+    moreInfoLink: 'https://support.terra.bio/hc/en-us/articles/360029071251-Google-Cloud-quotas-What-are-they-and-how-do-you-request-more-',
+    tooltip: 'Delayed by Google Cloud Platform (GCP) quota limits. Contact Terra Support to request a quota increase.'
+  },
+  unknown: {
+    id: 'unknown',
+    label: rawStatus => `Unexpected status (${rawStatus})`,
+    icon: style => icon('question', { size: iconSize, style: { color: colors.dark(), ...style } })
+  }
+}
+
+
+/**
+ * Collapses submission or workflow status.
+ *
+ * @param {string} rawStatus
+ * @returns {Object} one of `statusType.succeeded`, `statusType.failed`, `statusType.running`, or `statusType.submitted`
+ */
+export const collapseStatus = rawStatus => {
+  switch (rawStatus) {
     case 'Succeeded':
-      return 'succeeded'
+      return statusType.succeeded
     case 'Aborting': // only on submissions not workflows
     case 'Aborted':
     case 'Failed':
-      return 'failed'
+      return statusType.failed
     case 'Running':
-      return 'running'
+      return statusType.running
     default:
-      return 'submitted'
+      return statusType.submitted
   }
 }
 
-
-export const collapseCromwellExecutionStatus = status => {
-  switch (status) {
+/**
+ * Collapses Cromwell status, taking into account both execution and backend status values.
+ *
+ * @param {string} executionStatus from metadata
+ * @param {string} backendStatus from metadata
+ * @returns {Object} one of `statusType.succeeded`, `statusType.failed`, `statusType.running`, `statusType.waitingForCloudQuota`, or `statusType.unknown`
+ */
+export const collapseCromwellStatus = (executionStatus, backendStatus) => {
+  switch (executionStatus) {
     case 'Done':
-      return 'succeeded'
+      return statusType.succeeded
     case 'Aborting':
     case 'Aborted':
     case 'Failed':
-      return 'failed'
+      return statusType.failed
     case 'Running':
-      return 'running'
-    case 'Submitted':
```

Comment:
Removed `Submitted` because there is no Cromwell `Submitted` status, and this caused confusion when trying to understand the code. Here are the possible Cromwell executionStatuses: https://github.com/broadinstitute/cromwell/blob/17203758fa4608fdfb9e77194f67579b67343e39/core/src/main/scala/cromwell/core/ExecutionStatus.scala#L11-L26